src/EmitJs.re

```diff
@@ -1326,9 +1326,9 @@ let emitTranslationAsString =
            )
       : env;
 
-  let emitters = variantTables |> emitVariantTables(~config, ~emitters);
+  let moduleItems = ResolvedName.emitAllModuleItems();
```

Comment:
Keeping global state in `ResolvedName` is a way to test quickly, but would break if one were to process more than one file at once.
Better if the state is created explicitly at the caller site.
E.g.
```
let moduleItemsEmitter = ResolvedName.createModuleItemsEmitter ()

let moduleItems = ResolvedName.emitAllModuleItems(~moduleItemsEmitter);
...

```

```